pybamm/geometry/battery_geometry.py

```diff
@@ -70,6 +70,39 @@ def battery_geometry(
                         }
                     )
 
+    if options is not None and options["PE degradation"] == "phase transition":
+        geo_domain = geo.domain_params["positive"]
+        phases = int(getattr(options, domain)["particle phases"])
+        if phases == 1:
+            geometry.update(
+                {
+                    "positive core": {"r_co": {"min": 0, "max": geo_domain.prim.R_typ}},
```

Comment:
I was expecting to have the core/shell be solved on [0, 1] and then have transformed PDEs that solve for the lengths to handle the moving boundary. Can you point to the paper or some notes for the implementation?